### PR TITLE
OHRM5X-1124: Add i18n to pim module screens

### PR DESCRIPTION
**Pull request by SharukaNP**

### CODE REVIEW

**Comment by RajithaKumara** [comment]
Seems some commits are duplicated here from #1153. Rebase this PR once merge #1153

**Comment by SharukaNP** [comment]
> Seems some commits are duplicated here from #1153. Rebase this PR once merge #1153

Yeah su

> Seems some commits are duplicated here from #1153. Rebase this PR once merge #1153

Yeah sure

**Comment by SharukaNP** [comment]
@RajithaKumara Instead of re-base I pulled from the i18n where the admin related 1151 commits had been merged and solved the conflicts

File: installer/upgrader/Migrations/V5/generalLangString.yaml [hunk 1]
```diff
@@ -209,3 +209,336 @@ langStrings:
...
+    value: 'Attachments'
+    unitId: attachments
+  -
+    value: 'File Name'
+    unitId: file_name
+  -
+    value: 'Size'
+    unitId: size
+  -
+    value: 'Accepts jpg, .png, .gif up to 1MB. Recommended dimensions: 200px X 200px'
```
Comment (RajithaKumara) [inline installer/upgrader/Migrations/V5/generalLangString.yaml:384]:
```suggestion
    value: 'Accepts jpg, .png, .gif up to {size}MB. Recommended dimensions: 200px X 200px'
```

File: installer/upgrader/Migrations/V5/pimLangString.yaml [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+    value: 'Data Import'
+    unitId: data_import
+  -
+    value: 'Column order should not be changed'
+    unitId: column_order_should_not_be_changed
+  -
+    value: 'First Name and Last Name are compulsory'
+    unitId: first_name_and_last_name_are_compulsory
+  -
+    value: 'All date fields should be in YYYY-MM-DD format'
```
Comment (RajithaKumara) [inline installer/upgrader/Migrations/V5/pimLangString.yaml:48]:
```suggestion
    value: 'All date fields should be in {format} format'
```

File: installer/upgrader/Migrations/V5/pimLangString.yaml [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+    unitId: all_date_fields_should_be_in_yyyy_mm_dd_format
+  -
+    value: 'If gender is specified, value should be either Male or Female'
+    unitId: gender_specified_value_should_be_either_m_or_f
+  -
+    value: 'Each import file should be configured for 100 records or less'
```
Comment (RajithaKumara) [inline installer/upgrader/Migrations/V5/pimLangString.yaml:54]:
```suggestion
    value: 'Each import file should be configured for {count} records or less'
```

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**